site/src/Layers.js

```diff
@@ -9,6 +9,13 @@ export const layers = [
     activeColor: "hsla(17, 17%, 92%, 1)",
     labelColor: "hsla(0, 0%, 0%, 0)",
   },
+  {
+    theme: "base",
+    type: "bathymetry",
+    outline: true,
```

Comment:
The bathymetry layer is exclusively polygons. I chose an outline over a fill layer because the inner area of the bathymetry polygons overlap with land, leading to irrelevant clicks. Outline layers better represent boundaries.